### ISSUE TITLE
Unescaped left brace in regex is illegal here in regex

### ISSUE THREAD

**Issue by eserte** [sshaw/Mojolicious-Plugin-ParamLogger#2]
Tests fail with bleadperl (e.g. perl 5.25.4):

```
Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/debug: GET / { <-- HERE }/ at t/basic.t line 45.
# Looks like your test exited with 255 before it could output anything.
t/basic.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 17/17 subtests 
```

**Comment by jkeenan** [comment]
Applying a patch like the following should help:
```diff
$ diff t/basic.t.orig t/basic.t
45,47c45,47
< $t->get_ok('/')->content_like(qr|debug: GET / {}|, 'no params logged');
< $t->get_ok('/?a=1&b=2')->content_like(qr|debug: GET / { "a" => 1, "b" => 2 }|, 'multiple params logged');   
< $t->get_ok('/?password=p@ss')->content_like(qr|debug: GET / { "password" => "########" }|, 'password filtered');
---
> $t->get_ok('/')->content_like(qr|debug: GET / \{}|, 'no params logged');
> $t->get_ok('/?a=1&b=2')->content_like(qr|debug: GET / \{ "a" => 1, "b" => 2 }|, 'multiple params logged');   
> $t->get_ok('/?password=p@ss')->content_like(qr|debug: GET / \{ "password" => "########" }|, 'password filtered');
50c50
< $t->get_ok('/?a=1')->content_like(qr|debug: GET / { "a" => "########" }|, 'single param filtered');
---
> $t->get_ok('/?a=1')->content_like(qr|debug: GET / \{ "a" => "########" }|, 'single param filtered');
53c53
< $t->get_ok('/?a=1&b=2&c=3')->content_like(qr|debug: GET / { "a" => "########", "b" => "########", "c" => 3 }|, 'multiple params filtered');
---
> $t->get_ok('/?a=1&b=2&c=3')->content_like(qr|debug: GET / \{ "a" => "########", "b" => "########", "c" => 3 }|, 'multiple params filtered');
56c56
< $t->get_ok('/')->content_like(qr|info: GET / {}|, 'log level');
---
> $t->get_ok('/')->content_like(qr|info: GET / \{}|, 'log level');
59c59
< $t->get_ok('/')->content_unlike(qr|info: GET / {}|, 'disabled for production');
---
> $t->get_ok('/')->content_unlike(qr|info: GET / \{}|, 'disabled for production');
62c62
< $t->get_ok('/')->content_like(qr|info: GET / {}|, 'enabled for production');
---
> $t->get_ok('/')->content_like(qr|info: GET / \{}|, 'enabled for production');
```

**Comment by sshaw** [comment]
Thanks, fixed in [v0.02](https://github.com/sshaw/Mojolicious-Plugin-ParamLogger/tree/v0.02).
